lib-android-robolectric/src/main/java/com/devinci/lib/robolectric/AttributeSetBuilder.java

```diff
@@ -0,0 +1,38 @@
+package com.devinci.lib.robolectric;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import java.util.ArrayList;
+import java.util.List;
+import javax.annotation.Nonnull;
```

Comment:
Unfortunately, I couldn't get support-annotations to work with java project. It didn't resolve this dependency. I will try once again if there's some other way.